sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,16 +1,13 @@
 # Release History
-## 3.1.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 3.1.2 (2022-12-05)
```

Comment:
Should we add a section for 3.1.3 now?

Reply:
Does CI do that automatically for us when releasing?

Reply:
yes there should be a PR out by azure-sdk bot for that. If I don't see one, I can create another PR. But I want to avoid doing in same PR bcz it confuses customers.